README.md

```diff
@@ -54,12 +73,18 @@ In this phase everyone is thinking like a Designer.  Design thinking also relies
 
 ***Q1.5 Based on the feedback, how might you change your solution?***
 
+For our phone case, we would make it "smarter," meaning that it would monitor the amount of radiation and respond accordingly. More specifically, it would vibrate when the levels get above a certain threshold. It could also close apps to lower the radiation levels. For the Bluetooth headphones, we would likely change the design in order to make them more comfortable.
```

Comment:
You could even have a portion of the "Settings" app in your phone that allows you to set the average radiation output you want to maintain.